ui/page/components/wallet_setup_page.go

```diff
@@ -491,7 +528,7 @@ func (pg *CreateWallet) createWallet() {
 	walletName := pg.walletName.Editor.Text()
 	pass := pg.passwordEditor.Editor.Text()
 	seedType := GetWordSeedType(pg.seedTypeDropdown.Selected())
-	switch *pg.assetTypeSelector.SelectedAssetType() {
+	switch getAssetType(pg.assetTypeDropdown.Selected()) {
```

Comment:
This function call is not necessary, make `pg.assetTypeDropdown.Selected()` return an object of type `libutils.AssetType`.